jumpscale/clients/stellar/stellar.py

```diff
@@ -573,8 +590,15 @@ def _transfer(
         my_keypair = stellar_sdk.Keypair.from_secret(self.secret)
         transaction.sign(my_keypair)
 
-        response = horizon_server.submit_transaction(transaction)
-        tx_hash = response["hash"]
+        tx_hash = ""
+        if asset_code in _NETWORK_KNOWN_TRUSTS[self.network.value]:
+            if fund_transaction and asset_code == "TFT":
```

Comment:
the two if's should be reversed, and not known trusts, but conditions from the tx funding service

Reply:
checks updated https://github.com/threefoldtech/js-sdk/pull/3022/commits/29f545e3079918e8823baa8c99f166540afe53fd#diff-42adb1af219b45002b16f95f78c441b70a4c972e2affb5090f2f4d046ec7a999R596